### PR TITLE
Unapologetic Diona Nerfs: Part 2 of 8274

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Dionae can no longer slip being restrained for free.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

File: code/modules/mob/living/carbon/human/diona_gestalt.dm [hunk 1]
```diff
@@ -222,6 +222,10 @@
 	set desc = "Allows you to detach specific nymph, and control it."
 	set category = "Abilities"
 
+	if(stat || paralysis || stunned || weakened || lying || restrained())
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/diona_gestalt.dm]: we have `use_check_and_message` for this

**Comment by Yonnimer** [comment]
An alternative is being worked on where as a dionae will have a 30 timer that can be interrupted by moving them when they try to split a nymph off, and. 30 second timer to reattach the limb. This fixes the same issue of being able to instantly uncuff while also being the generally more liked option where we discussed this. In the mean time I’d request this be closed while that pr is being made. It’s incredibly shitty to ignore all the feedback on the idea in favor of just making yours.

File: code/modules/mob/living/carbon/human/diona_gestalt.dm (hunk 1 above)
Comment (ParadoxSpace) [reply code/modules/mob/living/carbon/human/diona_gestalt.dm]: Handled, thank you.

**Comment by ParadoxSpace** [comment]
Update?

**Pull request merged**